src/index.js

```diff
@@ -22,11 +23,34 @@ const HoneybadgerVue = {
         file: file
       }
     }
+    const shouldLogError = () => {
+      if (Vue.config.warnHandler) {
```

Comment:
Log error if:
- `Vue.config.warnHandler` is defined
- `console` is available and (`Vue.config.debug` or **not** production build)